### PR TITLE
[MIG] stock_picking_import_serial_number: Migration to 15.0

### PR DESCRIPTION
**Pull request by ryanc-me**
No changes required

### CODE REVIEW

**Comment by rousseldenis** [comment]
@ryanc-me Still WIP ?

**Comment by ryanc-me** [comment]
@rousseldenis it's working, but the module author is adding some fixes/improvements to the 13.0 branch (#1065). I'm waiting for that PR so I can include the fixes here too

**Comment by ryanc-me** [comment]
@rousseldenis @sebalix would also very much appreciate a review here :)

Please see https://github.com/OCA/stock-logistics-workflow/pull/1063#issuecomment-1295653235

File: stock_picking_import_serial_number/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Picking Import Serial Numbers",
     "summary": "Import S/N from excel file for incoming pickings",
-    "version": "14.0.1.0.2",
+    "version": "15.0.1.0.2",
```
Comment (rousseldenis) [inline stock_picking_import_serial_number/__manifest__.py:6]: Could you reset to "15.0.1.0.0" ?

**Comment by pedrobaeza** [comment]
@ryanc-me can you perform the change to merge this?

**Review by Saran440 — APPROVED** [review]
Functional Test import S/N in receive :+1:

**Comment by pedrobaeza** [comment]
Superseded by #1392 as no answer.

**Pull request closed without merging**